jcl/src/java.base/share/classes/java/lang/String.java

```diff
@@ -3847,7 +3847,49 @@ static String valueOfCodePoint(int codePoint) {
 		}
 		return string;
 	}
-	/*[ENDIF] Java11*/
+
+	/**
+	 * Returns a string whose value is the concatenation of this string repeated
+	 * count times. 
+	 * 
+	 * @param count
+	 *            a positive integer indicating the number of times to be repeated
+	 * @return a string whose value is the concatenation of this string repeated count times
+	 * @throws IllegalArgumentException
+	 *             if the count is negative
+	 * @since 11
+	 */
+	public String repeat(int count) {
+		if (count < 0) {
+			throw new IllegalArgumentException();
+		} else if (count == 0 || isEmpty()) {
+			return new String();
```

Comment:
Should this be a new String every time, or return "", which is the identical empty string every time.

Reply:
After determining the correct answer, add the case to the test suite.